backend/conf/logger.js

```diff
@@ -1,2 +1,39 @@
-const logger = require('simple-node-logger').createSimpleLogger('project.log');
-module.exports = logger;
\ No newline at end of file
+const simpleNodeLogger = require('simple-node-logger');
+
+const eventTypes = {
+  request: '[REQUEST]',
+  server: '[SERVER]',
+  app: '[APP]'
+}
+
+const logger = simpleNodeLogger.createSimpleLogger({ logFilePath: 'project.log', timestampFormat: 'YYYY-MM-DD - HH:mm:ss.SSS |' });
+
+logger.serverInfo = (message) => {
+  logger.info(`${eventTypes.server} ${message}`);
+};
+
+logger.requestInfo = (message) => {
+  logger.info(`${eventTypes.request} ${message}`);
+};
+
+logger.appInfo = (message) => {
+  logger.info(`${eventTypes.app} ${message}`);
+};
+
+logger.appWarn = (message) => {
+  logger.warn(`${eventTypes.app} ${message}`);
+};
+
+const requestMiddleware = (req, res, next) => {
+  const updateMethods = ['POST', 'PATCH'];
+
+  if(updateMethods.includes(req.method)) {
+    logger.requestInfo(`${req.method} ${req.originalUrl} ${JSON.stringify(req.body)}`);
+  } else {
+    logger.requestInfo(`${req.method} ${req.originalUrl}`);
+  }
```

Comment:
creo que podemos simplificarlo a un 
"Imprimí body si hay" directamente, 

```suggestion
    logger.requestInfo(`${req.method} ${req.originalUrl} ${  req.body ?? JSON.stringify(req.body)   }`);```